ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -1355,7 +1451,7 @@ public Table getTable(TableName tableName) throws HiveException {
    */
   public Table getTable(final String dbName, final String tableName,
                         boolean throwException) throws HiveException {
-    return this.getTable(dbName, tableName, throwException, false);
+    return this.getTable(dbName, tableName, throwException, true);
```

Comment:
Why is this change? Should not this minimally depend on if this table is transactional or not? What happens if we call it on a non-transactional table?

Reply:
This is not even related to DDL, perhaps a mistake in the original patch ? I am reverting for now.